packages/schema-blocks/src/functions/presenters/InnerBlocksSidebar.tsx

```diff
@@ -0,0 +1,128 @@
+import { useSelect } from "@wordpress/data";
+import { BlockInstance } from "@wordpress/blocks";
+import { __ } from "@wordpress/i18n";
+
+import { SvgIcon } from "@yoast/components";
+
+import { createAnalysisMessages, SidebarWarning } from "./SidebarWarningPresenter";
+import { ClientIdValidation, YOAST_SCHEMA_BLOCKS_STORE_NAME } from "../redux";
+import BlockSuggestions from "./BlockSuggestionsPresenter";
+import { BlockValidationResult } from "../../core/validation";
+
+interface InnerBlocksSidebarProps {
+	currentBlock: BlockInstance;
+	recommendedBlocks: string[];
+	requiredBlocks: string[];
+}
+
+/**
+ * Retrieves the validation results for the block with the given client ID from the Redux store.
+ *
+ * @param clientId The client ID of the block to retrieve the validation results for.
+ *
+ * @returns The validation results.
+ */
+function useValidationResults( clientId: string ): BlockValidationResult {
+	return useSelect( select => {
+		const results: ClientIdValidation = select( YOAST_SCHEMA_BLOCKS_STORE_NAME ).getSchemaBlocksValidationResults();
+		if ( ! results ) {
+			return null;
+		}
+
+		return results[ clientId ];
+	}, [ clientId ] );
+}
+
+/**
+ * Retrieves the latest block version from the WordPress store.
+ *
+ * @param clientId The client ID of the block to retrieve the latest version of.
+ *
+ * @returns The latest version of the block.
+ */
+function useBlock( clientId: string ): BlockInstance {
+	return useSelect( select => select( "core/block-editor" ).getBlock( clientId ), [ clientId ] );
+}
+
+/**
+ * Inner blocks sidebar component.
+ *
+ * @param props The properties.
+ *
+ * @returns The inner blocks sidebar component.
+ *
+ * @constructor
+ */
+export function InnerBlocksSidebar( props: InnerBlocksSidebarProps ): ReactElement {
+	const block = useBlock( props.currentBlock.clientId );
+	const validationResults = useValidationResults( props.currentBlock.clientId );
+
+	let warnings: SidebarWarning[] = [];
+
+	if ( validationResults ) {
+		warnings = createAnalysisMessages( validationResults );
+	}
+
+	return <Fragment>
+		<WarningList warnings={ warnings } />
+		<BlockSuggestions
+			title={ __( "Required Blocks", "yoast-schema-blocks" ) }
+			block={ block }
+			suggestions={ props.requiredBlocks }
+		/>
+		<BlockSuggestions
+			title={ __( "Recommended Blocks", "yoast-schema-blocks" ) }
+			block={ block }
+			suggestions={ props.recommendedBlocks }
+		/>
+	</Fragment>;
+}
+
+interface WarningListProps {
+	warnings: SidebarWarning[];
+}
+
+/**
+ * Renders a ReactElement containing the list of warnings.
+ *
+ * @param props The properties.
+ *
+ * @returns A ReactElement containing the list of warnings.
+ */
+function WarningList( props: WarningListProps ): ReactElement {
+	return (
+		<div className="yoast-block-sidebar-warnings">
+			<div className="yoast-block-sidebar-title">{ __( "Analysis", "yoast-schema-blocks" ) }</div>
+			<ul className="yoast-block-sidebar-warnings"> { }
```

Comment:
You forgot to remove the `{ }`

```suggestion
			<ul className="yoast-block-sidebar-warnings">
```